Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.5.1
+
+RUN apt-get update && apt-get install -y build-essential libffi-dev \
```

Comment:
The best practices document suggests using: --no-install-recommends to keep the size of the resultant images as small as possible.